### PR TITLE
COMMONAPACHELOG: match email address in auth field

### PR DESCRIPTION
**Pull request by mattjbray**
#3 matches email addresses in the `ident` field.  Also allow email addresses in the `auth` field.

### CODE REVIEW

**Comment by untergeek** [comment]
Please add a test (or better yet, tests) to https://github.com/logstash-plugins/logstash-patterns-core/tree/master/spec/patterns to guarantee that usage will work as expected and not break other patterns.

**Comment by mattjbray** [comment]
@untergeek I've added a couple of tests for this change and for the change in #3.

**Comment by purbon** [comment]
please jenkins, test this.

File: spec/patterns/httpd_spec.rb [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+  it "matches an email address in the ident field" do
+    expect(subject).to match("1.2.3.4 an.email@address.com - [22/Sep/2015:14:39:21 +0000] \"GET /index.html\" 200 182 \"https://my.website.com/index.html\" \"Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36\"")
+  end
+
+  it "matches an email address in the auth field" do
+    expect(subject).to match("1.2.3.4 - an.email@address.com [22/Sep/2015:14:39:21 +0000] \"GET /index.html\" 200 182 \"https://my.website.com/index.html\" \"Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36\"")
+  end
+
+end
```
Comment (purbon) [inline spec/patterns/httpd_spec.rb]: good additions to the test suites, I wonder if you could also add some that match the "former" USER pattern too? like this we'll be sure there are no regressions introduced.

**Comment by purbon** [comment]
@mattjbray thanks a lot for your contribution, I just left you a very small comment. As soon as we're good with it, I'm very welcome to merge this. Looking forward to it.

**Comment by mattjbray** [comment]
@purbon I've added a test for matching the USER pattern in both fields.

**Comment by purbon** [comment]
@mattjbray awesome, good job!

**Comment by purbon** [comment]
please jenkins, test this.

**Comment by jsvd** [comment]
this has been fixed in https://github.com/logstash-plugins/logstash-patterns-core/pull/142, thanks for the contribution!

**Pull request closed without merging**